src/alexapi/playback_handlers/vlchandler.py

```diff
@@ -69,7 +75,11 @@ def on_play(self, item):
 			vlcInstance = self.media_vlc_instance
 			player = self.media_player
 
-		media = vlcInstance.media_new(item.url)
+		if platform.system() == 'Windows':
```

Comment:
Use
```
		url = item.url
 		if platform.system() == 'Windows':
 			url = url.replace("file://", "")
```
it's cleaner and prevents mistakes when some branch of the condition might not define the variable.